scripts/app.js

```diff
@@ -1,35 +1,34 @@
 var allProjects = [];
-var allProjectString = [];
 
 function Project (opts) {
   this.title = opts.title;
-  this.author = opts.author;
+  this.language = opts.language;
   this.category = opts.category;
   this.gitHubUrl = opts.gitHubUrl;
   this.body = opts.body;
   this.publishedOn = opts.publishedOn;
 }
 $(document).ready(function(){
-  $(".projectClass").clone().appendTo(".projectClassSection");
-  $(".projectClass").clone().appendTo(".projectClassSection");
+  // var $newProject = $(".projectClass").clone().appendTo(".projectClassSection");
+  var projectStr = "";
+  for (var i = 0; i < allProjects.length; i++){
+    projectStr += "<article id='project1' class='projectClass'>" + allProjects[i].title + "<br>" +  allProjects[i].language + ", " + "<br>" +
+    allProjects[i].category + '<br>' + allProjects[i].publishedOn + "<br>" + ", " + "<br>" + allProjects[i].gitHubUrl + "<br>" + "<" + '/' + 'article' + ">" + "<br>";
+    $(".projectClassSection").append(projectStr);
+    projectStr = "";
+  }
 });
 
 Project.prototype.toHtml = function () {
-  // var $newProject = $("#projectSection").clone().apendTo("body");
-
-  // $('article.projectClass').clone();
-  // $newProject.data('category', this.category);
-  // $newProject.find('h1').text(this.title);
-  // return $newProject;
+  //   var $newProject = $(".projectClass").clone();
+  // $('section.projectClassSection').removeClass("projectClassSection");
```

Comment:
So you are not targeting the correct element to remove. You just created a variable (use it) that is actually what you need to target, so there is one fix there.